### PR TITLE
Adding PHP 8 support

### PR DESCRIPTION
**Pull request by tezhm**
Depends on #70

Updates dependencies to include PHP 8 support.

### CODE REVIEW

**Comment by tezhm** [comment]
Hey, I'm not sure how to update the build strategy to only build the 2.8.0 orm with PHP 8.

**Comment by eigan** [comment]
@tezhm try this:
```diff
    name: PHP ${{ matrix.php }} - doctrine/orm ${{ matrix.orm }}
    strategy:
      matrix:
        php: ['7.2', '7.3', '7.4', '8.0']
        orm: ['2.6.0', '2.7.0', '2.8.0']
+      exclude:
+        - php: '8.0'
+          orm: 2.6.0'
+        - php: '8.0'
+          orm: 2.7.0'
    steps:
      - uses: actions/checkout@v2
      - name: Setup PHP

```

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -6,8 +6,13 @@ jobs:
...
       matrix:
-        php: ['7.2', '7.3', '7.4']
-        orm: ['2.6.0', '2.7.0']
+        php: ['7.2', '7.3', '7.4', '8.0']
+        orm: ['2.6.0', '2.7.0', '2.8.0']
+      exclude:
+        - php: '8.0'
+          orm: '2.6.0'
+        - php: '8.0'
+          orm: '2.7.0'
```
Comment (eigan) [inline .github/workflows/ci.yml]: Should be with same indention as "orm". Take a look here:

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-excluding-configurations-from-a-matrix

My bad, sorry, hehe :)

Comment (tezhm) [reply .github/workflows/ci.yml]: Ahhh, thankyou. See how the next build goes.

Comment (eigan) [reply .github/workflows/ci.yml]: Try to allow phpunit 8 for PHP 7.2 support.

Comment (tezhm) [reply .github/workflows/ci.yml]: Whoops, was testing the composer require step locally and committed those changes.

**Comment by eigan** [comment]
Looks good @tezhm! Thanks for this and #70 👍🏻

**Comment by tezhm** [comment]
No worries man, glad to help out.

**Comment by eigan** [comment]
Should be released with 1.2.1 now @tezhm. Thanks!

**Pull request merged**